tests/packagedcode/data/pypi/requirements.txt/gemfileparser-requirements-txt-expected.json

```diff
@@ -0,0 +1,52 @@
+{
+  "type": "pypi",
+  "namespace": null,
+  "name": null,
+  "version": null,
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Python",
+  "description": null,
+  "release_date": null,
+  "parties": [],
+  "keywords": [],
+  "homepage_url": null,
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": null,
+  "copyright": null,
+  "license_expression": null,
+  "declared_license": null,
+  "notice_text": null,
+  "root_path": null,
+  "dependencies": [
+    {
+      "purl": "pkg:pypi/setuptools",
+      "requirement": "==32.0.0",
+      "scope": "dependencies",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false
+    },
+    {
+      "purl": "pkg:pypi/nose",
+      "requirement": "==1.3.7",
```

Comment:
See my comments about the case of `==` pinned requirements ... there we should also add that to the pulr version and mark this as resolved